main/midnight/src/system/moonring.cpp

```diff
@@ -146,10 +160,24 @@ void moonring::continueStory( storyid_t id )
     // start game
     if (!checkGameOverConditions() ) {
         character& c = TME_CurrentCharacter();
-        auto mode = Character_IsDead(c) ? MODE_THINK : MODE_LOOK;
         
+        if ( Character_IsDead(c) ) {
+            TME_SelectChar(c.id);
+            showPage( MODE_THINK );
+            return true;
```

Comment:
Heads up, compile error here. I will create a PR for this branch, to include also replacement of DIS by RES.

Reply:
Ok. Thanks. I'll be interested to see why it baulked. That looked pretty inoffensive. :)